cellengine/utils/api_client/APIClient.py

```diff
@@ -120,6 +129,53 @@ def update_entity(self, experiment_id, _id, entity_type, body) -> dict:
             json=body,
         )
 
+    def _get_path(self, entity: CE) -> str:
+        fullpath = f"{self.base_url}/{entity.path}"
+        return fullpath
+
+    def create(self, entity: CE, **kwargs) -> CE:
+        """Create a local entity on CellEngine."""
```

Comment:
What's the plan for all of the existing APIClient methods, like `post_compensation`?

Reply:
I think they should remain, as they are still useful. I needed `.create` for Gates, and I figured I may as well allow all objects to be created this way.